src/pet.js

```diff
@@ -0,0 +1,68 @@
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = 10;
+    this.children = [];
+}
+
+Pet.prototype = {
+    get isAlive() {
+        return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+}
+
+Pet.prototype.growUp = function() {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+    };
+    const ADD_HUNGER = 5;
+    const TAKE_FITNESS = 3;
```

Comment:
Good idea adding constants, though it would be better if they were all defined in one place rather than scattered about the whole file. To go one better, you could put them in their own file, meaning you could then import it into this file as well as the test files too, this would mean you wouldnt need to use specific values in your test cases, instead just referring to the constants. 
If you dont do this, and say the value for maximum hunger changes, you would have to change hard coded values used in multiple files. While that isnt that big of a deal in a small project like this, in an enterprise level app with millions of lines you can see how this could easily get out of control if an eye isnt kept on it :)